src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -550,8 +551,8 @@ export default {
           legalName: this.entityName
         }
       }
-
-      if (this.hasFilingCode(FilingCodes.ADDRESS_CHANGE_OT) && this.addresses) {
+      if (this.hasFilingCode(this.isBComp ? FilingCodes.ADDRESS_CHANGE_BC
```

Comment:
Consider making the ternary expression a getter here as well.

Reply:
Copy that, will do 👍

Reply:
Updated in next commit